.gitmodules

```diff
@@ -1,6 +1,10 @@
-[submodule "src/Common_FPGA_Modbus"]
-	path = src/Common_FPGA_Modbus
-	url = https://github.com/lsst-ts/Common_FPGA_Modbus
-[submodule "src/Common_FPGA_HealthAndStatus"]
-	path = src/Common_FPGA_HealthAndStatus
-	url = https://github.com/lsst-ts/Common_FPGA_HealthAndStatus
+[submodule "src/Common_FPGA_Modbus"]
+	path = src/Common_FPGA_Modbus
+	url = https://github.com/lsst-ts/Common_FPGA_Modbus
+[submodule "src/Common_FPGA_HealthAndStatus"]
+	path = src/Common_FPGA_HealthAndStatus
+	url = https://github.com/lsst-ts/Common_FPGA_HealthAndStatus
+[submodule "src/Modbus_Processing_Unit"]
+	path = src/Modbus_Processing_Unit
+	url = https://github.com/lsst-ts/Modbus_Processing_Unit
+	branch = tickets/DM-31196
```

Comment:
It might be better to change back to the `master` or `main` branch in a latter time.